hugo/content/docs/grammar-language.md

```diff
@@ -478,14 +487,24 @@ terminal ML_COMMENT: /\/\*[\s\S]*?\*\//;
 ``` 
 
 #### Negated Token
-It is possible to negate all tokens using the operator `!`, in regular expression the operator `^` is used for negation within a character class.
+It is possible to negate tokens using the operator `!`. In Xtext, this allows recognizing the *complement* of a set of characters (or anything 'but' what is listed in the negation), very much akin to a negated character class in regular expressions. For example, the following terminal would recognize zero or more characters between two # symbols.
 ```
 terminal BETWEEN_HASHES:'#' (!'#')* '#';
 ```
-In regular expression:
+
+This also works as expected when you use alternation in Xtext, `!('#'|'@')`, to recognize anything but `#` or `@`.
+
+*However, this works differently in Langium!* Instead of recognizing characters that are *not* in the set, it instead produces a negative lookahead. I.e., it does *not* consume tokens, but instead is a 'guard' for what the following expression can recognize. If you want to recognize a word that doesn't start with `no`, then you could write it in EBNF like so:
```

Comment:
```suggestion
*However, this works differently in Langium!* Instead of recognizing characters that are *not* in the set, it produces a negative lookahead. I.e., it does *not* consume tokens, but it is a 'guard' for what the following expression can recognize. If you want to recognize a word that doesn't start with `no`, then you could write it in EBNF like so:
```